### PR TITLE
Update user map for Nov 2016 #6203

### LINKED ISSUE #6203: Update user map for Nov 2016
Here is the list of institutions to add to the user map.
```
Australia	Curtin University of Technology
Australia	Management Alternatives
Australia	Murdoch University
Belgieum	Annuntia
Belgium	Bacaputsteen
Brasil	Associação Educacional Dom Bosco
Brazil	AEDB
Brazil	PUC Minas
Canada	Royal Roads University
Canada	Ryerson University
Canada	Sheridan College
Canada	Simon Fraser University
Canada	University of Ivory Tower
Canada	University of Lethbridge
Canada	University of Ontario
Canada	University of Ottawa
Croatia	University of Zagreb
France	EDHEC
France	EM Strasbourg
France	IESEG
France	INSEAD
France	Telecom Paris Tech
Germany	EP Marktplatz
HK	HKU
Hungary	Codecool
Indonesia	Islamic University of Malang
Iraq	GCC Services
Ireland	Dublin City University
Italy	University of Padua
Lebanon	Lebanese American University
Malaysia	Help University
Malaysia	Sunway University
Malaysia	Universiti Teknologi Malaysia
Malaysia	University of Malaya
Malaysia	University of Reading
metrocity	BITS
Mexico	ITESM
Mexico	Universidad de Celaya
Netherlands	Avans Hogeschool
Netherlands	Delft University of Technology
Netherlands	Maerlant
Netherlands	Maris College Belgisch Park
Netherlands	Saxion University of Applied Sciences
New Zealand 	University of Auckland 
Norway	Høgskolen i Østfold
Poland	Codecool Poland
Saudi Arabia	King Fahd University of Petroleum and Minerals
Singapore	Nanyang Technological University
Singapore	Singapore Institute of Technology
Singapore	Singapore Polytechnic
Singapore	SUTD
South Africa 	Maritzburg College
Spain	University of Deusto
Sweden	International School of the Stockholm Region
Sweden	Örebro University
Taiwan	National University of Kaohsiung
UAE	Hult International Business School
UAE	Zayed University
UK	City University London
UK	Royal Holloway University of London
UK	Strathclyde University
UK	UCLU Malaysian Society 
UK	University College London
UK	University of Birmingham
UK	University of Southampton
UK	University of Westminster
USA	Alcorn  
USA	Baylor University
USA	Bearden High School
USA	Bedford North Lawrence High School
USA	Brigham Young University, Idaho
USA	Buffalo ISD
USA	Butte College
USA	California State University, Fullerton
USA	Camden Middle School
USA	Christopher Newport University
USA	Columbus State Community College
USA	CSU Chico
USA	Des Moines Area Community College
USA	Discovery High School
USA	Dominican University of California
USA	Douglas S. Freeman High School
USA	Drexel University
USA	Emory University
USA	Gallaudet University
USA	Grand Valley State University
USA	Indiana University Bloomington
USA	International Council of Malaysian Scholars and Associates (ICMS)
USA	Iowa State University 
USA	Lee University
USA	Macomb Community College
USA	Marquette University
USA	Midwives College of Utah
USA	Montclair State University
USA	New Mexico State University
USA	New Mexico State University
USA	Northeastern University
USA	Ohio University 
USA	Olivia Nieto-Herrera Elementary
USA	Oxford College of Emory University
USA	Southern Wesleyan University
USA	St. Anne Community High School 302
USA	Tarrant County College
USA	Tennessee Tech University
USA	Texas Woman's University 
USA	The Hun School of Princeton
USA	The Shipley School
USA	UC Berkeley
USA	University of Arizona
USA	University of California, Berkeley
USA	University of Cincinnati
USA	University of Maryland, Baltimore County
USA	University of Minnesota
USA	University of New Mexico
USA	University of Wisconsin, La Crosse
USA	Virginia Commonwealth University
USA	Washington University
USA	Widener University
```

### PR DESCRIPTION
**Pull request by hackosinn**
Fixes #6203

### CODE REVIEW

File: src/main/webapp/js/userMapData.json [hunk 1]
```diff
@@ -384,15 +417,21 @@
     "Universiti Tunku Abdul Rahman",
     "University College of Technology Sarawak",
     "University Kuala Lumpur",
-    "University of Nottingham Malaysia Campus"
+    "University of Malaya",
+    "University of Nottingham Malaysia Campus",
+    "University of Reading"
   ],
+  "metrocity": [
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is not a proper country name, so you can exclude it entirely.

File: src/main/webapp/js/userMapData.json [hunk 2]
```diff
@@ -425,6 +466,7 @@
     "Rotterdam School of Management",
     "Rotterdam University of Applied Science",
     "Saxion University",
+    "Saxion University of Applied Sciences",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as "Saxion University" but since it is the more proper name you can delete that one instead.

**Comment by hackosinn** [comment]
@wkurniawan07 - why merge branch into patch-3 failed?  will the failed check pose a problem?

**Comment by wkurniawan07** [comment]
@hackosinn apologies for that, we were experimenting with AppVeyor and it seems like this PR gets a collateral damage. You can ignore that failure; another push should remove it entirely.
Anyway, you also need to change `usermap.jsp` and `README.md`. Essentially you have to do what is done in #6318 here instead of in another PR/another branch. Do take note as well that November only has 30 days.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # TEAMMATES Developer Web Site
 
-![Users Count](https://img.shields.io/badge/users-80K%2B-brightgreen.svg)
-![Universities Count](https://img.shields.io/badge/universities-1000%2B-green.svg)
+![Users Count](https://img.shields.io/badge/users-140K%2B-brightgreen.svg)
```
Comment (wkurniawan07) [inline README.md]: Please do not change this line (users count) as it is not related to this issue.

**Comment by wkurniawan07** [comment]
@hackosinn for your subsequent contributions if you wish to, please include the issue number in the issue title (as shown now after I edited it), and name your branch following the convention mentioned in [this document](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#creating-a-branch) instead of `patch-n`.

**Comment by hackosinn** [comment]
@wkurniawan07 - Sir i messed up with my forked branch if i delete the repository and again fork the teammates, will my contribution lost?
I want to set up teammate again on my laptop. Plzz reply

**Pull request merged**